### PR TITLE
Add support for Docker in DO and StaticVirtualMachine

### PR DESCRIPTION
**Pull request by maxking**
* Indent "pass" and anything else under a class with 4
  space to properly indent a multiline class name or
  PEP8's E125 error creeps up!
* Adds support for ContainerizedVirtualMachine to
  DigitalOcean and StaticVirtualMachine

### CODE REVIEW

**Comment by maxking** [comment]
I signed it!

**Comment by voellm** [comment]
This looks good to me.

Klaus can you double check?  Thanks.

File: perfkitbenchmarker/digitalocean/digitalocean_virtual_machine.py [hunk 1]
```diff
@@ -237,11 +237,17 @@ def CreateScratchDisk(self, disk_spec):
     self.scratch_disks.append(digitalocean_disk.DigitalOceanDisk(disk_spec))
 
 
+class ContainerziedDigitalOceanVirtualMachine(
```
Comment (klausw) [inline perfkitbenchmarker/digitalocean/digitalocean_virtual_machine.py]: Spelling: s/Containerzied/Containerized/

**Comment by klausw** [comment]
Looks good aside from the typo. Thank you for the contribution, and sorry about the long delay in responding.

FYI, in general it's preferable not to combine whitespace changes with other changes since it confuses the history. It's ok in this case since it's a small change, no need to split it out.

**Comment by klausw** [comment]
I've cherry-picked this into the master branch and fixed the typo. Thanks again!

**Comment by maxking** [comment]
Thanks for the tips! I will take care about that next time.

**Pull request merged**